DailyKit/CallManager/Models/CallParticipant.swift

```diff
@@ -0,0 +1,76 @@
+import Daily
+
+extension VideoTrack: Hashable {
+    public func hash(into hasher: inout Hasher) {
+        hasher.combine(id)
+        hasher.combine(isEnabled)
+    }
+}
+
+public struct CallParticipant: Hashable, Identifiable {
+    enum Constants {
+        static let defaultUsername: String = "Guest"
+    }
+
+    public let id: AnyHashable
+    public internal(set) var username: String
+    public let isLocal: Bool
+    public let hasAudio: Bool
+    public let videoTrack: VideoTrack?
+    public var videoScaleMode: VideoView.VideoScaleMode { isLocal ? .fit : .fill }
+    public let hasVideo: Bool
+}
+
+extension CallParticipant {
+    init(_ participant: Participant) {
+        self.id = participant.id
+        self.username = participant.info.username ?? Constants.defaultUsername
+        self.isLocal = participant.info.isLocal
+        self.hasAudio = participant.media?.microphone.state == .playable
+        let videoTrack = participant.media.flatMap { media in
```

Comment:
Personally I prefer not destructing types like that most of the time. I've worked on many projects where consistent naming isn't followed, which then makes using search and replace for refactoring really hard because you can't search for consistent property names. You have to find the property, figure out what it was renamed to as a local, then search for that, and you have to do that at every call site. Xcode doesn't have great refactoring support compared to some other editors, so that probably influences my perspective.

I don't mind changing it, but since this is a nit, I'd prefer to not introduce locals here.